aws/logs_monitoring/parsing.py

```diff
@@ -401,8 +403,13 @@ def parse_service_arn(source, key, bucket, context):
             elbname = name.replace(".", "/")
             if len(idsplit) > 1:
                 idvalue = idsplit[1]
-                return "arn:aws:elasticloadbalancing:{}:{}:loadbalancer/{}".format(
-                    region, idvalue, elbname
+                partition = "aws"
```

Comment:
Can this be generic or only really needed for elb?

Reply:
Added parsing for redshift as well. We can't handle gov partitions for keys that don't contain the region, using the lambda arn / region environment var wouldn't work since customers can have a lambda in their gov account which is forwarding events from a commercial source.